app/frontend/src/features/map/MapPage.tsx

```diff
@@ -31,6 +34,11 @@ export default function MapPage() {
   const initializeMap = (map: MapboxMap) => {
     map.on("load", () => {
       addClusteredUsersToMap(map, handleClick);
+      map.on("click", (ev: any) => {
+        console.log(ev.lngLat.lat, ev.lngLat.lng);
```

Comment:
`console.log` is usually not left in committed code.

Reply:
Yes, this is debugging code.